src/util/LanguageUtil.js

```diff
@@ -19,6 +19,7 @@ function loadLanguage(appState, settings) {
   }, supportedLanguages).then(function() {
     clearTimeout(timeout);
     appState.trigger('loading', false);
+    appState.trigger('removeLoading');
```

Comment:
`trigger('loading', false)` will do nothing here because of this line: 
https://github.com/okta/okta-signin-widget/blob/97e8b741cb8825449e92ba46bd9684dd58625b21/src/v1/views/shared/Header.ts#L180-L181

We need to trigger `removeLoading` event. There are similar approaches in v1 codebase. Example
https://github.com/okta/okta-signin-widget/blob/97e8b741cb8825449e92ba46bd9684dd58625b21/src/v1/controllers/RecoveryLoadingController.js#L31-L33